app/controllers/spaces_controller.rb

```diff
@@ -32,6 +32,6 @@ def destroy
   end
 
   def space_params
-    params.require(:space).permit(:location, :volume, :condition, :access, :price_per_month, :user, :description, :image)
+    params.require(:space).permit(:location, :volume, :condition, :access, :price_per_month, :user, :description, :photo)
```

Comment:
be aware that it is called `:image` everywhere on the master. if you change it to photo, you will have to ensure that this is done for every other file wherever this may be used, including mine or Charline's.

Reply:
As far as I know I changed it because it didnt work with image for whatever reason